language/bytecode-verifier/bytecode-verifier-tests/src/unit_tests/bounds_tests.rs

```diff
@@ -100,7 +100,7 @@ fn invalid_locals_id_in_call() {
         type_parameters: SignatureIndex::new(1),
     });
     let func_inst_idx = FunctionInstantiationIndex(m.function_instantiations.len() as u16 - 1);
-    m.function_defs[0].code.code = vec![CallGeneric(func_inst_idx)];
+    m.function_defs[0].code.as_mut().unwrap().code = vec![CallGeneric(func_inst_idx)];
```

Comment:
The pattern "as_mut()/as_ref().unwrap()" appears a times in the verifier. They are probably fine as we don't seem to run the abstract interpreter passes on a native function. But still this irks me and I'll see if I can clean up the code a bit and get rid of them.